cli/plasmo/src/commands/init.ts

```diff
@@ -75,9 +75,11 @@ async function init() {
   )
 
   const packageFilePath = resolve(projectDirectory, "package.json")
+  const packageNameFromDirectory = paramCase(currentDirectory.split('/').pop())
```

Comment:
`currentDirectory.split('/').pop()` is not cross-platform. We should use https://nodejs.org/api/path.html#pathbasenamepath-ext instead